tfx/components/example_validator/executor.py

```diff
@@ -63,30 +66,41 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
       output_dict: Output dict from key to a list of artifacts, including:
         - output: A list of 'ExampleValidationPath' artifact of size one. It
           will include a single pbtxt file which contains all anomalies found.
-      exec_properties: A dict of execution properties. Not used yet.
+      exec_properties: A dict of execution properties.
+        - exclude_splits: Names of splits that the example validator should not
+          validate.
 
     Returns:
       None
     """
     self._log_startup(input_dict, output_dict, exec_properties)
 
-    absl.logging.info('Validating schema against the computed statistics.')
-    label_inputs = {
-        labels.STATS:
-            tfdv.load_statistics(
-                io_utils.get_only_uri_in_dir(
-                    artifact_utils.get_split_uri(input_dict[STATISTICS_KEY],
-                                                 'eval'))),
-        labels.SCHEMA:
-            io_utils.SchemaReader().read(
-                io_utils.get_only_uri_in_dir(
-                    artifact_utils.get_single_uri(input_dict[SCHEMA_KEY])))
-    }
-    output_uri = artifact_utils.get_single_uri(output_dict[ANOMALIES_KEY])
-    label_outputs = {labels.SCHEMA_DIFF_PATH: output_uri}
-    self._Validate(label_inputs, label_outputs)
-    absl.logging.info(
-        'Validation complete. Anomalies written to {}.'.format(output_uri))
+    exclude_splits = exec_properties[EXCLUDE_SPLITS_KEY]
+    schema = io_utils.SchemaReader().read(
+        io_utils.get_only_uri_in_dir(
+            artifact_utils.get_single_uri(input_dict[SCHEMA_KEY])))
+    for artifact in input_dict[STATISTICS_KEY]:
+      for split in artifact_utils.decode_split_names(artifact.split_names):
+        if exclude_splits and split in exclude_splits:
```

Comment:
exclude_splits won't be None now, but keep this exclude_splits is no harm, up to you

Reply:
i think this is good convention to confirm the list we are accessing is not none? if this is too extra, i can take this out